webapp/components/CommentCard/CommentCard.vue

```diff
@@ -0,0 +1,216 @@
+<template>
+  <base-card v-if="isUnavailable" class="comment-card">
+    <p>
+      <base-icon name="ban" />
+      {{ this.$t('comment.content.unavailable-placeholder') }}
+    </p>
+  </base-card>
+  <base-card v-else :class="commentClass" :id="anchor">
+    <header class="header">
+      <user-teaser :user="comment.author" :date-time="comment.createdAt">
+        <template v-if="wasEdited" #dateTime>
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-02-12T12:41:03Z" title="Wednesday, February 12th 2020, 1:41:03 pm +01:00">Feb 12, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -0,0 +1,216 @@
+<template>
+  <base-card v-if="isUnavailable" class="comment-card">
+    <p>
+      <base-icon name="ban" />
+      {{ this.$t('comment.content.unavailable-placeholder') }}
+    </p>
+  </base-card>
+  <base-card v-else :class="commentClass" :id="anchor">
+    <header class="header">
+      <user-teaser :user="comment.author" :date-time="comment.createdAt">
+        <template v-if="wasEdited" v-slot:dateTime>
```


```suggestion
        <template v-if="wasEdited" #2117 dateTime>
```
I think this is copy :spaghetti: , but consider using shorthand, as it's in line with the docs you wrote https://github.com/Human-Connection/Human-Connection/blob/bbd678caa6215865df78aff90de4a518d5d39136/webapp/vue.md#we-use-shorthands
See, https://vuejs.org/v2/guide/components-slots.html#Named-Slots-Shorthand

Reply:
> [<img alt="alina-beck" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/alina-beck) **Authored by [alina-beck](https://github.com/alina-beck)**
_<time datetime="2020-02-17T10:00:15Z" title="Monday, February 17th 2020, 11:00:15 am +01:00">Feb 17, 2020</time>_

---


I saw you already created a bounty issue for this – great idea! :+1: – so I just replaced the `v-slots` I added in this PR with the shorthand and left the other instances for our bounty hunters... you got me there. 😜